### PR TITLE
Monster's IDs for the resources

### PR DESCRIPTION
**Pull request by Malckyor**

### CODE REVIEW

File: docs/resources/Monster.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Monster Resource
+
+Monsters ID's here.
+
+###### Monster IDs
+
+|           ID          | Hex |
+| ----------------------|-----|
+|  UNK                  | 00  |
```
Comment (Tsugami) [inline docs/resources/Monster.md:9]: what is this?

File: docs/resources/Monster.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Monster Resource
+
+Monsters ID's here.
+
+###### Monster IDs
+
+|           ID          | Hex |
```
Comment (Tsugami) [inline docs/resources/Monster.md:7]:
```suggestion
|           MONSTER HUNTER          |  HEX |
```

File: docs/resources/Monster.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+|  Monoblos             | 1A  |
+|  Velocidrome          | 1B  |
+|  Gendrome             | 1C  |
+|  Rocks                | 1D  |
+|  Ioprey               | 1E  |
+|  Iodrome              | 1F  |
+|  Pugis                | 20  |
+|  Kirin                | 21  |
+|  Cephalos             | 22  |
+|  Giaprey / Giadrome   | 23  |
```
Comment (Tsugami) [inline docs/resources/Monster.md:44]: I think this is wrong, other "preys" have separate ids, here it shouldn't be different

File: docs/resources/Monster.md (hunk 1 above)
Comment (Malckyor) [reply docs/resources/Monster.md:9]: I don't know, I never actually saw this in any quest nor never used it.

Comment (Tsugami) [reply docs/resources/Monster.md:9]: better open an issue when i merge over it

File: docs/resources/Monster.md (hunk 3 above)
Comment (Tsugami) [reply docs/resources/Monster.md:44]: I will also open a issue for this

File: docs/resources/Monster.md (hunk 2 above)
Comment (Tsugami) [reply docs/resources/Monster.md:7]: fix in #3

**Comment by Tsugami** [comment]
Thank very much for your contribution, I hope we can make this repository very useful for everyone ❤️

**Pull request merged**